homu/init.sls

```diff
@@ -44,3 +44,9 @@ homu:
     - require:
       - pip: homu
       - file: /home/servo/homu/cfg.toml
+
+toml:
+  pip.installed:
+    - pkgs:
+      - toml == 0.9.1  # Ensure this is up to date with requirements.txt
```

Comment:
Instead of a separate Salt state (block), add this to the `pip.installed` state above that installed Homu. Also, let's say 'in sync' instead of 'up to date'.